packages/tupaia-web/src/features/Map/Map.tsx

```diff
@@ -63,16 +66,24 @@ const TilePickerWrapper = styled.div`
   }
 `;
 
+const ENTITY_FIELDS = ['parent_code', 'code', 'name', 'type', 'bounds', 'region'];
```

Comment:
Technically, this doesn't need to be it's own variable because it's only used in one place, but I also understand that it feels tidier. Just a thought.

Reply:
yes that's right I just pulled it out for readability. I feel like it might end up being a shared const soon or I might create a couple of versions of the useEntities hook.